src/adapter/src/catalog.rs

```diff
@@ -2133,12 +2133,28 @@ impl CatalogItem {
         }
     }
 
-    pub fn initial_logical_compaction_window(&self) -> Option<Duration> {
-        let custom_logical_compaction_window = match self {
+    pub fn custom_logical_compaction_window(&self) -> Option<Duration> {
```

Comment:
Can you add doc-comments to these two functions. It's not immediately obvious what the difference is.